src/httk/atomistic/structureutils.py

```diff
@@ -946,8 +946,12 @@ def get_primitive_basis_transform(hall_symbol):
 #     # Transform to primitive cell
 #     return lattrans
 
-def transform(structure, transformation, max_search_cells=20, max_atoms=1000):
+def transform(structure, transformation, max_search_cells=20, max_atoms=5000, use_hall_symbol = None):
+    """Applies a transformation matrix to the structure
 
+    Args:
+        use_hall_symbol (bool, optional): Enforces a supplied hall_symbol. If True, defaults into using P 1.
```

Comment:
@Abhivega Just a quick tought before I merge this; is perhaps `force_hall_symbol` a better name than `use_hall_symbol` for this? Given that things probably will break if you give the wrong one, I feel that name better highlights that you need to know what you are doing if you use that parameter.